cobigen-templates/templates-devon4j/src/main/templates/crud_angular_client_app/templates/src/app/${variables.etoName#lower_case}/${variables.etoName#lower_case}-grid/${variables.etoName#lower_case}-grid.component.scss

```diff
@@ -1,3 +1,51 @@
-.td-expansion-panel-header {
-    background-color: #cccccc;
+mat-card-title {
+  padding-top: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
```

Comment:
Maybe there is a reason behind this, but pixels are usually not a good idea. Depending on the screen density, it may look different. 

I think we should use percentages, do you agree @sjimenez77 ?

Reply:
I checked it in a few of the devices in Chrome's device emulator. Visually it looks alright.

If we switch to percentages we need to calculate the correct percentage for different screen dimensions and use media queries to implement it. Seems too much work for a small thing. Should I proceed anyhow?

Reply:
I see. I think that as we are on a hurry we should proceed without fixing it, it is a nice to have feature.